architecture/additional_concepts/service_catalog.adoc

```diff
@@ -0,0 +1,321 @@
+[[architecture-additional-concepts-service-catalog]]
+= Service Catalog
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[service-catalog-overview]]
+== Overview
+
+[NOTE]
+====
+The service catalog is currently a Technology Preview feature.
+ifdef::openshift-origin,openshift-enterprise[]
+To opt-in during installation or upgrade, see Installation & Configuration (TODO link).
+endif::[]
+====
+
+When developing microservices-based applications to run on cloud native
+platforms, there can be many ways to provision different resources and share
+their coordinates (URL),  credentials, and configuration, depending on the
+service provider and the platform.
+
+To give developers a more seamless experience, {product-title} includes a
+_service catalog_, an implementation of the
+link:https://openservicebrokerapi.org/[Open Service Broker API] (OSB API) for
+Kubernetes. This allows users to connect any of their applications deployed in
+{product-title} to a wide variety of service brokers in a microservices-based
```

Comment:
I'm not convinced that the scope here is solely microservices - I'd be inclined to delete `in a microservices-based environment`.